test/repos.js

```diff
@@ -17,12 +18,12 @@ describe('repos', () => {
     expect(repos.express).to.equal('https://github.com/expressjs/express')
   })
 
-  it('is always a URL', function () {
-    this.timeout(10 * 1000)
+  it('is always a URL or null', function () {
+    this.timeout(30 * 1000)
     const urls = Object.values(repos)
-    urls.forEach(url => {
-      expect(isUrl(url), `${url}`).to.eq(true)
-    })
+
+    expect(urls.some(url => isUrl(url)), 'should contain packages with repos').to.equal(true)
+    expect(urls.some(url => url === null), 'should contain packages without repos').to.equal(true)
```

Comment:
This is expecting at least a `null`, which will not be the case if this is merged into current master. I'd say use the test to expect what we know it might be there, either a null or an url..

```js
for (const url of Object.values(repos)) {
  expect(url === null || isUrl(url), url).to.equal(true)
}
```

Also nothing prevents the packages to be all set, and while super low probability there could be no-null at some point :))

Reply:
>which will not be the case if this is merged into current master

Agree, it will fail if we merge it without updating the data. On the other hand master with these changes and old index should be considered broken so probably it's fine.

>while super low probability there could be no-null at some point

The probability of that is so low that I would rather consider it a bug if our index doesn't have any `null` value. It will prevent the situation when packages without a repository are missing in the index because of a bug in the code.